zkevm-circuits/src/root_circuit/aggregation.rs

```diff
@@ -613,6 +613,16 @@ pub mod test {
             layouter.assign_region(
                 || "",
                 |mut region| {
+                    // annotation columns
```

Comment:
By checking the source code I see that the circuit `StandardPlonk` is just a test circuit used to generate proofs and then aggregate them with the AggregationCircuit (via the `TestAggregationCircuit` type).
The `StandardPlonk` is never checked with the `MockProver` so I think these annotations are not needed (and are not annotations to the RootCircuit).

The RootCircuit that would require annotations is `RootCircuit` defined in `zkevm-circuits/src/root_circuit.rs` and the region it uses seems to be defined in `zkevm-circuits/src/root_circuit/aggregation.rs` by `"Aggregate snarks"`.
The `RootCircuit` uses the `MainGate` and `Range`, and it's their associated columns that should be annotated.
I checked the `MainGateConfig` and it has a method that returns the advices, but no method to return the fixed columns, so I think we can only do partial annotation (unless we update `halo2wrong` to expose the fixed columns).
As for the `RangeConfig` I think we're less lucky and it doesn't expose any of its columns.

Reply:
Thanks! I was hesitant to add annotation to a test circuit or not. I thought it's fine because it's a 'test' circuit.

fixed in [c48e4d8](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1173/commits/c48e4d8e90fa6af01d3113cbbdeabd88312b3721)